synapse/federation/federation_client.py

```diff
@@ -173,7 +174,7 @@ def backfill(self, dest, context, limit, extremities):
 
     @defer.inlineCallbacks
     @log_function
-    def get_pdu(self, destinations, event_id, outlier=False):
+    def get_pdu(self, destinations, event_id, outlier=False, timeout=None):
         """Requests the PDU with given origin and ID from the remote home
         servers.
```

Comment:
Some doc-string explaining how the timeout is used and what units it is in might be nice.